Lib/test/test_exceptions.py

```diff
@@ -322,19 +323,18 @@ def baz():
     @unittest.skipIf(ctypes.sizeof(ctypes.c_int) >= ctypes.sizeof(ctypes.c_ssize_t),
                      "Downcasting to int is safe for col_offset")
     @support.requires_resource('cpu')
-    @support.bigmemtest(2**(ctypes.sizeof(ctypes.c_int)*8-1)-1-len("pass"), memuse=1)
+    @support.bigmemtest(INT_MAX, memuse=2, dry_run=False)
     def testMemoryErrorBigSource(self, size):
-        if size < 2**(ctypes.sizeof(ctypes.c_int)*8-1)-1-len("pass"):
+        padding_needed = INT_MAX-len("pass")
+        if size < padding_needed:
```

Comment:
`size` is always INT_MAX, so this condition is always false.

Reply:
Oh, that's what `dry_run=False` does?